src/types/app-client.ts

```diff
@@ -1425,21 +1416,22 @@ export class AppClient {
   }
 
   /** Make the given call and catch any errors, augmenting with debugging information before re-throwing. */
-  private async handleCallErrors<TResult>(call: () => Promise<TResult>) {
-    try {
-      return await call()
-    } catch (e) {
-      const logicError = await this.exposeLogicError(e as Error)
-      if (logicError instanceof LogicError) {
-        let currentLine = logicError.teal_line - logicError.lines - 1
-        const stackWithLines = logicError.stack
-          ?.split('\n')
-          .map((line) => `${(currentLine += 1)}: ${line}`)
-          .join('\n')
-        Config.logger.error(`${logicError.message}\n\n${stackWithLines}`)
-      }
-      throw logicError
-    }
+  private handleCallErrors = async (e: Error) => {
+    // Only handle errors for this app.
+    const appIdString = `app=${this._appId.toString()}`
```

Comment:
How reliable is this?

Reply:
Continuing this discussion in main thread for visibility